sale_stock_picking_note/models/sale_stock.py

```diff
@@ -11,6 +12,15 @@ class SaleOrder(models.Model):
         string="Picking Note",
     )
 
+    @api.onchange(
+        'partner_id',
+    )
+    def onchange_partner_picking_note(self):
+        partner = self.partner_id
+        if partner:
+            partner_picking_note = partner.default_picking_note
```

Comment:
@SirTakobi Is the real desired behavior of deleting the existing note if there is already one defined (and maybe lose data)?

Reply:
I think it is, but it seems to me that you are implying it's a bad thing :smile:
@eLBati can you please check if what's happening is the desired behavior?